python/test/test_ingest_stream.py

```diff
@@ -0,0 +1,106 @@
+import json
+
+from timeplus import Stream, Query
+import datetime
+
+
+def test_ingest(test_environment):
+    name = "test_ingest_stream"
+    data = ["time", "data"]
+    values = [[datetime.datetime.now(), "abcd"]]
+
+    stream = (
```

Comment:
you can create a reference stream for ingest as well, dont have to include the creation of the stream in the test code